packages/workorder-angular/package.json

```diff
@@ -13,7 +13,7 @@
     "url": "https://github.com/feedhenry-raincatcher/raincatcher-angularjs.git"
   },
   "author": "feedhenry-raincatcher@redhat.com",
-  "license": "MIT",
+  "license": "Apache 2.0",
```

Comment:
I think the right string is `Apache-2.0`, see https://spdx.org/licenses/Apache-2.0.html

With this one I get `npm WARN @raincatcher/angularjs-workorder@1.0.0 license should be a valid SPDX license expression`